src/main/resources/application.properties

```diff
@@ -1,9 +1,7 @@
-spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.url=jdbc:postgresql://localhost:5432/SprintManager
 spring.datasource.username=postgres
 spring.datasource.password=123
-spring.jpa.generate-ddl=true
-
 
 debug=true
```

Comment:
Do we need these properties: `debug` and `server.port` ?

Reply:
> Do we need these properties: `debug` and `server.port` ?
standart port 8080 is already used by postgres, thats why i changed it